hippo4j-spring-boot/hippo4j-config-spring-boot-1x-starter/src/main/java/cn/hippo4j/config/springboot1x/starter/config/WebThreadPoolHandlerConfiguration1x.java

```diff
@@ -47,6 +48,7 @@ public class WebThreadPoolHandlerConfiguration1x {
 
     @Configuration
     @ConditionalOnClass({Servlet.class, Tomcat.class})
+    @ConditionalOnBean(WebThreadPoolRunStateHandler.class)
```

Comment:
Why do you need to add this condition?

Reply:
Actually a defense, Spring container startup fails when the WebThreadPoolRunStateHandler class is not successfully created (hippo4j.enable=false).